packages/notification-center/src/components/notification-center/components/AppContent.tsx

```diff
@@ -29,7 +29,7 @@ export function AppContent() {
 
 const wrapperClassName = (primaryColor: string, fontFamily: string, dir: string) => css`
   margin: 0;
-  font-family: ${fontFamily}, Helvetica, sans-serif;
```

Comment:
Not sure if it is a good thing to remove the fallback font families. If it doesn't stop for our users to add the font family they chose if it can be loaded, I would leave the fallbacks.

Reply:
added the fallbacks 🚀